lib/utils/src/mime.rs

```diff
@@ -236,21 +260,25 @@ fn test_basic() {
     assert!(MIME::str_parse("image/png;immortal=true").is_ok());
     assert!(MIME::str_parse("image/png;immortal=true;mutant[]=2,3,4,5").is_ok());
     assert!(MIME::str_parse("image/png;immortal=true;mutant[]=2,3,4,5")
-        .map_err(|_| "mutant verify_param")
+        .map_err(|_| "mutant str_parse")
         .unwrap()
         .verify_param(
             b"image/png;immortal=true;mutant[]=2,3,4,5",
             "mutant[]",
             b"2,3,4,5"
-        ));
-    assert!(MIME::str_parse("image/png;immortal=true;mutant[]=2,3,4,5")
+        )
         .map_err(|_| "mutant verify_param")
+        .unwrap());
+    assert!(MIME::str_parse("image/png;immortal=true;mutant[]=2,3,4,5")
+        .map_err(|_| "mutant str_parse")
         .unwrap()
         .verify_param(
             b"image/png;immortal=true;mutant[]=2,3,4,5",
             "immortal",
             b"true"
-        ));
+        )
+        .map_err(|_| "mutant verify_param")
+        .unwrap());
```

Comment:
These changes may conflict with https://github.com/sporeprotocol/spore-contract/pull/28/commits/1e100f6a4cc74c4da3eec076e2cce235b438cbff in your another PR.

Again, it is better to solve one small issue in one PR.
For now, it LGTM.

Reply:
yes, next step is to solve conflict, the time is ok